src/components/Card/__stories__/Card.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react';
+import {Card, CardProps} from '../Card';
+import {CardShowcase} from './CardShowcase';
+
+import './Card.stories.scss';
+
+export default {
+    title: 'Components/Card',
+    component: Card,
+} as Meta;
+
+const DefaultTemplate: Story<CardProps> = (args: any) => (
+    <Card {...args} className="card-stories" />
```

Comment:
Maybe `.card-stories` should be a container? Or that example card should have content to be styled, not the component itself.

Reply:
I've refactored stories 6c24cf8